content/en/getting_started/tracing/_index.md

```diff
@@ -37,10 +37,23 @@ vagrant ssh
 
 To install the Datadog Agent on a host, use the [one line install command][6] updated with your [Datadog API key][7]:
 
+
+{{< site-region region="us" >}}
+
+```shell
+DD_API_KEY=<DATADOG_API_KEY>  bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
+```
+
+{{< /site-region >}}
+
+{{< site-region region="eu" >}}
+
 ```shell
-DD_API_KEY=<DATADOG_API_KEY> bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
+DD_API_KEY=<DATADOG_API_KEY> DD_SITE="datadoghq.eu" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
 ```
 
+{{< /site-region >}}
```

Comment:
can we also add US1-FED instructions? otherwise nothing will show up when looking at the US1-FED version of this page

Reply:
Oooooh, good call. Keep forgetting about the ol' US1-FED. Will update!